common/src/main/java/com/wynntils/models/gear/type/AbstractItemInfoDeserializer.java

```diff
@@ -0,0 +1,416 @@
+        GearRestrictions restrictions = parseRestrictions(json);
+        ItemMaterial material = parseMaterial(json, type);
+
+        List<ItemObtainInfo> obtainInfo = parseObtainInfo(json, name);
+
+        Optional<StyledText> loreOpt = parseLore(json);
+        Optional<String> apiNameOpt = Optional.ofNullable(apiName);
+
+        boolean allowCraftsman = JsonUtils.getNullableJsonBoolean(json, "allowCraftsman");
+
+        return new GearMetaInfo(
+                dropRestrictions, restrictions, material, obtainInfo, loreOpt, apiNameOpt, allowCraftsman);
+    }
+
+    protected List<ItemObtainInfo> parseObtainInfo(JsonObject json, String name) {
+        List<ItemObtainInfo> obtainInfo = new ArrayList<>();
+
+        // Add crowd-sourced information
+        List<ItemObtainInfo> obtainCrowdSourced = Models.WynnItem.getObtainInfo(name);
+        obtainInfo.addAll(obtainCrowdSourced);
+
+        // Add API-obtained information
+        String apiObtainName =
+                JsonUtils.getNullableJsonString(JsonUtils.getNullableJsonObject(json, "dropMeta"), "name");
+        List<ItemObtainType> apiObtainTypes = parseObtainTypes(json);
+        for (ItemObtainType apiObtainType : apiObtainTypes) {
+            switch (apiObtainType) {
+                case NORMAL_MOB_DROP -> {
+                    // Only add this if we do not have more specific information
+                    boolean hasSpecialMob =
+                            obtainInfo.stream().anyMatch(o -> o.sourceType() == ItemObtainType.SPECIAL_MOB_DROP);
+                    if (!hasSpecialMob) {
+                        obtainInfo.add(new ItemObtainInfo(ItemObtainType.NORMAL_MOB_DROP, Optional.empty()));
+                    }
+                }
+                case DUNGEON_RAIN -> {
+                    // Only add this if we do not have more specific information
+                    boolean hasDungeon =
+                            obtainInfo.stream().anyMatch(o -> o.sourceType().isDungeon());
+                    if (!hasDungeon) {
+                        obtainInfo.add(new ItemObtainInfo(ItemObtainType.DUNGEON_RAIN, Optional.empty()));
+                    }
+                }
+                default -> {
+                    // Add the API-obtained information
+                    obtainInfo.add(new ItemObtainInfo(apiObtainType, Optional.ofNullable(apiObtainName)));
+                }
+            }
+        }
+
+        if (obtainInfo.isEmpty()) {
+            // We have no information on how to obtain this
+            obtainInfo.add(new ItemObtainInfo(ItemObtainType.UNKNOWN, Optional.empty()));
+        }
+
+        return List.copyOf(obtainInfo);
+    }
+
+    protected Optional<StyledText> parseLore(JsonObject json) {
+        String lore = JsonUtils.getNullableJsonString(json, "lore");
+        if (lore == null) return Optional.empty();
+
+        // Some lore contain like "\\[Community Event Winner\\]", fix that
```

Comment:
Just noticed this: Is this still relevant in the v3 api?

Reply:
Removed it, it's not.